network-formula/network/wicked/map.jinja

```diff
@@ -0,0 +1,30 @@
+{#-
+Jinja variables file for the Wicked Salt states
+Copyright (C) 2023 Georg Pfuetzenreuter <mail+opensuse@georg-pfuetzenreuter.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- from 'network/map.jinja' import interfaces, routes -%}
+{%- set interfaces = interfaces -%}
+{%- set routes = routes -%}
```

Comment:
`set interfaces = interfaces` and `set routes = routes` look like no-ops to me. Do they do anything useful I overlooked, or can they be removed?

Reply:
It's a Jinja caveat, variables created by `from` are not exported, hence including them from this map file will fail unless exported using `set`.

Reply:
Thanks, I wasn't aware of this detail.